compiler/esy.json

```diff
@@ -11,7 +11,8 @@
     "compile": "dune build @all @install",
     "clean": "rm -rf #{self.root}/_esy",
     "install-compiler": "dune install",
-    "test": "dune runtest --force"
+    "test": "dune runtest --force",
+    "format": "esy dune build @fmt --auto-promote"
```

Comment:
Do you think `format` should just be moved into the regular build command? I want a way to ensure that formatting always happens. Maybe it'd be a good idea too to have the CI fail if things aren't formatted.

Reply:
Yeah, there's a bunch of build and CI stuff to solve. I'm going to send another PR that tries to fix some of the stuff that has been popping up in the discord.